spec/controllers/selfservice/advisers_controller/create_spec.rb

```diff
@@ -23,7 +23,7 @@ module SelfService
         end
 
         it 'adds a success flash message' do
-          expect(flash[:notice]).to eq('Saved successfully')
+          expect(flash[:notice]).to eq(I18n.t('self_service.adviser_edit.saved'))
```

Comment:
Total stickler point here.  Any reason why you're using the 'I18n.t' notation here but the shorter 't' notation in the _alerts.erb file?